[2021-09-27 14:07:39,353] {taskinstance.py:903} INFO - Dependencies all met for <TaskInstance: example_complex.create_entry_gcs_result 2021-09-27T14:06:38.240034+00:00 [queued]>
[2021-09-27 14:07:39,417] {taskinstance.py:903} INFO - Dependencies all met for <TaskInstance: example_complex.create_entry_gcs_result 2021-09-27T14:06:38.240034+00:00 [queued]>
[2021-09-27 14:07:39,418] {taskinstance.py:1095} INFO - 
--------------------------------------------------------------------------------
[2021-09-27 14:07:39,419] {taskinstance.py:1096} INFO - Starting attempt 1 of 1
[2021-09-27 14:07:39,421] {taskinstance.py:1097} INFO - 
--------------------------------------------------------------------------------
[2021-09-27 14:07:39,440] {taskinstance.py:1115} INFO - Executing <Task(BashOperator): create_entry_gcs_result> on 2021-09-27T14:06:38.240034+00:00
[2021-09-27 14:07:39,444] {standard_task_runner.py:52} INFO - Started process 1362 to run task
[2021-09-27 14:07:39,453] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'example_complex', 'create_entry_gcs_result', '2021-09-27T14:06:38.240034+00:00', '--job-id', '12', '--pool', 'default_pool', '--raw', '--subdir', '/home/***/.local/lib/python3.6/site-packages/***/example_dags/example_complex.py', '--cfg-path', '/tmp/tmpne9469bz', '--error-file', '/tmp/tmpdwu7l8uq']
[2021-09-27 14:07:39,460] {standard_task_runner.py:77} INFO - Job 12: Subtask create_entry_gcs_result
[2021-09-27 14:07:39,593] {logging_mixin.py:109} INFO - Running <TaskInstance: example_complex.create_entry_gcs_result 2021-09-27T14:06:38.240034+00:00 [running]> on host fa8cc667bc0b
[2021-09-27 14:07:39,739] {taskinstance.py:1254} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=example_complex
AIRFLOW_CTX_TASK_ID=create_entry_gcs_result
AIRFLOW_CTX_EXECUTION_DATE=2021-09-27T14:06:38.240034+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2021-09-27T14:06:38.240034+00:00
[2021-09-27 14:07:39,747] {subprocess.py:52} INFO - Tmp dir root location: 
 /tmp
[2021-09-27 14:07:39,750] {subprocess.py:63} INFO - Running command: ['bash', '-c', 'echo create_entry_gcs_result']
[2021-09-27 14:07:39,768] {subprocess.py:74} INFO - Output:
[2021-09-27 14:07:39,774] {subprocess.py:78} INFO - create_entry_gcs_result
[2021-09-27 14:07:39,775] {subprocess.py:82} INFO - Command exited with return code 0
[2021-09-27 14:07:39,838] {taskinstance.py:1219} INFO - Marking task as SUCCESS. dag_id=example_complex, task_id=create_entry_gcs_result, execution_date=20210927T140638, start_date=20210927T140739, end_date=20210927T140739
[2021-09-27 14:07:39,905] {local_task_job.py:151} INFO - Task exited with return code 0
[2021-09-27 14:07:39,966] {local_task_job.py:261} INFO - 0 downstream tasks scheduled from follow-on schedule check
